src/sass/components/_tour.scss

```diff
@@ -37,12 +37,40 @@
 
 .popover-wrapper {
   position: absolute;
-  top: -10px;
+  top: -12px;
+
+  .epic-process & {
+    right: -10px;
+    top: -16px;
+  }
 
   .logo-popover & {
     top: unset;
   }
 
+  .metecho-breadcrumb & {
+    left: 0;
+    top: -15px;
+  }
+
+  .next-steps & {
+    left: 200px;
+  }
+
+  .project-placeholder & {
+    top: 15px;
+  }
+
+  .right-edge & {
+    right: 0;
+    top: -8px;
+  }
+
+  .vertical-right & {
+    right: -6px;
+    top: -20px;
+  }
+
   .page-title & {
     top: 25px;
```

Comment:
@dvdherron There are quite a few "magic numbers" in this file and I wonder if there is a more systematic way of positioning these? Maybe not, but it may be helpful to see some custom properties or variables https://www.lightningdesignsystem.com/design-tokens/#category-spacing

Reply:
@stacyk Made an attempt to clean these up with tokens. Since they are all being contained in different ways, it was tough to come up with a system that would work across the board. Let me know if this helps or not.